src/configure_settings/configure_settings.jl

```diff
@@ -23,6 +23,7 @@ function default_settings()
         "MethodofMorris" => 0,
         "IncludeLossesInESR" => 0,
         "EnableJuMPStringNames" => false,
+        "PieceWiseHeatRate" => 0
```

Comment:
Prefer `Piecewise` to `PieceWise` since it's a single word. https://en.wikipedia.org/wiki/Piecewise

Reply:
done!